interface-definitions/system-display.xml.in

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <!--
+    system display model (SDEC|EZIO)
+    system display config (enabled|disabled)
+    system display show host (cpu|cpu-all|cpu-hist|disk|load-hist|memory|proc|uptime)
+                        network interface <intName> alias <alias>
+                                units (bps|Bps|pps)
+                        clock (big|mini|date-time)
+                        title <name>
+
+    system display duration <s>
+    system display hello <string>
+    system display bye <string>
+    -->
+  <node name="system">
+    <children>
+      <node name="display" owner="${vyos_conf_scripts_dir}/system-display.py">
+        <properties>
+          <help>System display LCD/VFD/LED</help>
+          <priority>400</priority>
+        </properties>
+        <children>
+          <!-- system display model (SDEC|EZIO) -->
+          <leafNode name="model">
+            <properties>
+              <help>Model of the display attached to this system [REQUIRED]</help>
+              <completionHelp>
+                <list>SDEC EZIO</list>
+              </completionHelp>
+              <constraint>
+                <regex>(SDEC|EZIO)</regex>
+              </constraint>
+              <constraintErrorMessage>Invalid system display model</constraintErrorMessage>
+              <valueHelp>
+                <format>SDEC</format>
+                <description>Display model for Lanner, Watchguard, Nexcom NSA, Sophos UTM appliances with built-in SDEC LCD</description>
+              </valueHelp>
+              <valueHelp>
+                <format>EZIO</format>
+                <description>Display model for Portwell, Caswell appliances with built-in EZIO-100 or EZIO-300 LCD</description>
+              </valueHelp>
+            </properties>
+          </leafNode><!--system display model-->
+          <!-- system display config (enabled|disabled) -->
+          <leafNode name="config">
```

Comment:
I do not get the intention of this node here. So I can configure the display and then turn it off? If this is the case, we usually use a `disable` leafNode which is `<valueless/>`. We tend to no longer add in new boolean nodes which are enable/disable.

Reply:
What you are proposing was my intent. I was not aware of <valueless/>. I will do this.